code/modules/vending/_vending.dm

```diff
@@ -262,7 +262,12 @@
 			new /obj/item/stack/sheet/iron(loc, 3)
 		qdel(src)
 	else
-		..()
+		var/turf/T = get_turf(src)
+		if(T)
+			for(var/obj/item/I in contents)
+				I.forceMove(T)
+			explosion(src, devastation_range = -1, light_impact_range = 3)
```

Comment:
This is on ALL vending machines, is that intentional?

Reply:
Jesus, how tired was I when I wrote this?